cluster/ubuntu/util.sh

```diff
@@ -438,14 +438,19 @@ function kube-down {
 function prepare-push() {
   #Not yet support upgrading by using local binaries.
   if [[ $KUBE_VERSION == "" ]]; then
-     echo "Upgrading nodes to local binaries is not yet supported.Please specify the version"
-     exit 1
+    if [[ ! -d "ubuntu/binaries" ]]; then
+      echo "No local binaries.Please check"
+      exit 1
+    else 
+      echo "Please make sure all the required local binaries are prepared ahead"
+      sleep 3
+    fi
+  else
+    # Run build.sh to get the required release 
+    pushd ubuntu
+    source "build.sh"    
+    popd
```

Comment:
'Cause ubuntu/build.sh manages downloading binaries. Should use local ones instead in this scenario.